scripts/deploy-federation.sh

```diff
@@ -194,10 +194,14 @@ fi
 # Create a permissive clusterrolebinding to allow federation controllers to run.
 if [[ "${NAMESPACED}" ]]; then
   # TODO(marun) Investigate why cluster-admin is required to view cluster registry clusters in a namespace
-  kubectl -n "${NS}" create rolebinding federation-admin --clusterrole=cluster-admin --serviceaccount="${NS}:default"
+  if ! kubectl -n "${NS}" get rolebinding federation-admin; then
+      kubectl -n "${NS}" create rolebinding federation-admin --clusterrole=cluster-admin --serviceaccount="${NS}:default"
+  fi
 else
   # TODO(marun) Make this more restrictive.
-  kubectl create clusterrolebinding federation-admin --clusterrole=cluster-admin --serviceaccount="${NS}:default"
+  if ! kubectl get clusterrolebinding federation-admin; then
+      kubectl create clusterrolebinding federation-admin --clusterrole=cluster-admin --serviceaccount="${NS}:default"
+  fi
```

Comment:
are these unrelated to the rest of changes?

Reply:
This is an enhancement.  It could be removed if you dont like it .   For me , not like apply ,`kubectl create` would fail in reusing cluster.

Reply:
actually i like it.
but i guess it's easier to review if you separate unrelated changes, or at least mention them in the commit message.

Reply:
Please separate out unrelated changes.